### ISSUE TITLE
installation shows error

### ISSUE THREAD

**Issue by riccardoklinger** [heremaps/xyz-qgis-plugin#15]
I used the plugin installer to install the plugin in QGIS 3.4.4... Unfortunately it "fails" with this message:
```
  File "/home/ricckli/.local/share/QGIS/QGIS3/profiles/default/python/plugins/XYZHubConnector/modules/layer/__init__.py", line 11, in 
    from .layer import XYZLayer
  File "/usr/lib/python3/dist-packages/qgis/utils.py", line 672, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
  File "/home/ricckli/.local/share/QGIS/QGIS3/profiles/default/python/plugins/XYZHubConnector/modules/layer/layer.py", line 141
    sql_constraint = f'"{parser.QGS_XYZ_ID}" TEXT UNIQUE ON CONFLICT REPLACE' # replace older duplicate
```
My setup:
```
Python version: 3.5.2 (default, Nov 12 2018, 13:43:14) [GCC 5.4.0 20160609] 
QGIS version: 3.4.4-Madeira Madeira, f6ddc62 
```
On Ubuntu/Elementary OS

**Comment by minff** [comment]
fix in [version 1.6.2](/heremaps/xyz-qgis-plugin/releases/tag/1.6.2-alpha)

**Comment by riccardoklinger** [comment]
Yet you need to agree that the new LTR release is 3.4.8 which should every plugin be tested on. I was trying to change my Pythonpath to use the locally available 3.7.2 python version. yet no success. I would propose to check maybe the version:
`import sys
if sys.version_info.minor < 3.6:
blablub`
thanks for the change. works now on my system as well. would close this, when 1.6.2 is in the official repo.

**Comment by minff** [comment]
Regarding the python version, I think that depends on when the QGIS is built and the platform. 
For instance, my qgis builds (Windows) all comes with recent python version:
```
QGIS version 3.4.4-Madeira
Python: 3.7.0 (v3.7.0:1bf9cc5093, Jun 27 2018, 04:59:51) [MSC v.1914 64 bit (AMD64)]
```
```
QGIS version 3.0.3-Girona
3.6.0 (v3.6.0:41df79263a11, Dec 23 2016, 08:06:12) [MSC v.1900 64 bit (AMD64)]
```
So to have QGIS with recent python version, I guess we need to rebuild or redownload the preferred QGIS release.
